cerberus/api.py

```diff
@@ -125,31 +124,35 @@ def complete(self, request, pk=None):
         return self.change_state("complete", request)
 
     @action(detail=True, methods=["PUT"])
-    def move_booking(self, request, pk=None):
-        booking = self.get_object()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Verify serializer replacement maintains functionality.

Replacing 'BookingMoveSerializer' with 'ToDateSerializer' may affect data validation and parsing. Ensure that the new serializer adequately supports the required fields and validation rules for booking movement.